components/post_view/post_header/index.js

```diff
@@ -11,7 +11,7 @@ import PostHeader from './post_header.jsx';
 function mapStateToProps(state, ownProps) {
     return {
         ...ownProps,
-        displayNameType: get(state, Preferences.CATEGORY_DISPLAY_SETTINGS, 'name_format', 'false')
```

Comment:
Huh, I'm surprised no one noticed that this wasn't using the global setting, and was relying on the (previously) outdated preference